src/nginxconfig/templates/domain_sections/php.vue

```diff
@@ -219,6 +314,61 @@ THE SOFTWARE.
                 },
```

Comment:
You need to update this watcher to also disable/enable `phpServer` and `phpBackupServer`